### ISSUE TITLE
language import from `@inlang/sdk-js` is breaking switchLanguage function

### ISSUE THREAD

**Issue by NilsJacobsen** [inlang/inlang#938]
https://github.com/inlang/inlang/assets/58360188/4367b47c-dfeb-4f27-88af-753ad2e1c279

## Problem

When I import the language, it breaks the switchLanguage functionality.  Am I handling that correct?

```svelte
<script lang="ts">
	import { languages, switchLanguage, loadResource, language as selectedLanguage } from '@inlang/sdk-js';
</script>

<div class="relative bg-white w-fit border border-slate-300 rounded-xl py-1 px-[6px]">
	{#each languages as language}
		<button
			on:focus={() => loadResource(language)}
			on:mouseover={() => loadResource(language)}
			on:click={() => switchLanguage(language)}
		>
			<div class="relative py-1 px-3 text-md text-white z-20">
				{language}
			</div>
			
		</button>
	{/each}
	
	<div class={"absolute w-[43px] h-[30px] translate-y-[5px] bg-[#f73c02] top-0 rounded-md z-10 shadow-lg "}/>
</div>

{selectedLanguage}
```

**Comment by NilsJacobsen** [comment]
experienced while working on this: https://github.com/inlang/example/issues/24

**Comment by ivanhofer** [comment]
will be fixed in #797

**Comment by NilsJacobsen** [comment]
🚀

**Comment by ivanhofer** [comment]
The issue should be fixed in version `0.8.0`
